webserver/src/main/java/io/kestra/webserver/controllers/MiscController.java

```diff
@@ -67,7 +71,7 @@ public Configuration configuration() {
             .version(versionProvider.getVersion())
             .isTaskRunEnabled(executionRepository.isTaskRunEnabled())
             .isAnonymousUsageEnabled(this.isAnonymousUsageEnabled)
-            .isWorkerInstanceEnabled(false)
+            .isWorkerInstanceEnabled(workerHeartbeatRepositoryInterface.isPresent())
```

Comment:
This boolean is now useless as we will always have worker instance (except for the memory runner but we can live with that). You can remove it and always display the page.